bin/holdings_deleter.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+require "services"
+require "cluster"
+require "date"
+
+Services.mongo!
+
+STRING_FIELDS = [
+  "organization",
+  "local_id",
+  "enum_chron",
+  "n_enum",
+  "n_chron",
+  "status",
+  "condition",
+  "mono_multi_serial",
+  "country_code",
+  "uuid",
+  "issn"
+].freeze
+
+DATE_FIELDS  = ["date_received"].freeze
+INT_FIELDS   = ["ocn"].freeze
+FLOAT_FIELDS = ["weight"].freeze
+BOOL_FIELDS  = ["gov_doc_flag"].freeze
+CTRL_FIELDS  = ["noop", "verbose", "leave_empties"].freeze
+
+# Takes simple pairs of key=>value and constructs a update_many...pull query
+# based on those criteria.
+# Example:
+# bundle exec ruby bin/delete_matching_holdings.rb organization foo date_received 2020-01-01
+class HoldingsDeleter
+  attr_reader :matching_criteria
+
+  def initialize(args)
+    @matching_criteria = {}
```

Comment:
I think this can be more opinionated. Primary motivation for this is deleting holdings by org right? No need to make things flexible if we don't have other use cases.

Reply:
There is already need for doing this by `org+date_received`, and we're going to have to do it by `org+mono_multi_serial` (if the past is any indication of the future). At least. And at that point we'd have half a Swiss army knife.